frontend/src/views/HomePage/discover/Discover.tsx

```diff
@@ -6,7 +6,7 @@ import { getAuctions } from "store/auctions";
 import { useSearchParams } from "react-router-dom";
 import { DiscoverFormState } from "interfaces";
 
-const PRICE_GAP = 20;
+const PRICE_GAP = 50;
```

Comment:
question: what does PRICE_GAP do?

If determines the "increment" value on a price slider or something like this, you can set the `step` property on the input element, so that it snaps when the user slides the slider / clicks an arrow to increment the input value.